grafana/dashboard_test.go

```diff
@@ -123,6 +123,17 @@ func TestV5Dashboard(t *testing.T) {
 			So(dash.Panels[1].GridPos.H, ShouldEqual, 6)
 			So(dash.Panels[1].GridPos.W, ShouldEqual, 24)
 		})
+
+		Convey("Panels GridPos should convert int to float", func() {
+			So(dash.Panels[1].GridPos.H, ShouldEqual, 6)
```

Comment:
Since, these assertions are exactly the same as those on lines 123 and 124, I think you can just omit these two.

Reply:
y agree, I forgot to remove that :)
```suggestion
```

Reply:
y I agree, fixed in commit c7bb90d